scalding-core/src/main/scala/com/twitter/scalding/FileSource.scala

```diff
@@ -119,6 +110,32 @@ object AcceptAllPathFilter extends PathFilter {
 }
 
 object FileSource {
+  val LOG = LoggerFactory.getLogger(this.getClass)
```

Comment:
have you tested this on the cluster? Seems like serialization woes are possible here.